susemanager-utils/susemanager-sls/src/beacons/pkgset.py

```diff
@@ -16,9 +16,9 @@
 __virtualname__ = 'pkgset'
 
 PKG_PLUGINS = OrderedDict([
-        ("/usr/lib/zypp/plugins/commit/venv-zyppnotify", "/var/opt/venv-salt-minion/cache/rpmdb.cookie"),
-        ("/usr/share/yum-plugins/venv-yumnotify.py",     "/var/opt/venv-salt-minion/cache/rpmdb.cookie"),
-        ("/usr/bin/venv-dpkgnotify",                     "/var/opt/venv-salt-minion/cache/dpkg.cookie"),
+        ("/usr/lib/zypp/plugins/commit/venv-zyppnotify", "/var/venv-salt-minion/cache/rpmdb.cookie"),
+        ("/usr/share/yum-plugins/venv-yumnotify.py",     "/var/venv-salt-minion/cache/rpmdb.cookie"),
+        ("/usr/bin/venv-dpkgnotify",                     "/var/venv-salt-minion/cache/dpkg.cookie"),
```

Comment:
@vzhestkov: 
Would it be better to use path "/var/cache/venv-salt-minion/" instead of "/var/venv-salt-minion/cache/" ?
IMHO it would better fits to the standard of most packages or linux derivats.

Reply:
@Shirocco88 thanks for pointing it, it's a mistake for sure

Reply:
Fixed